coaster-nn/src/frameworks/native/mod.rs

```diff
@@ -820,6 +834,61 @@ impl<T> Pooling<T> for Backend<Native>
     }
 }
 
+impl<T> Rnn<T> for Backend<Native>
+    where T: Float + Default + Copy + PartialOrd + Bounded {
+    fn new_rnn_config(&self, src: &SharedTensor<T>, dropout_probability: Option<f32>, dropout_seed: Option<u64>, sequence_length: i32, network_mode: RnnNetworkMode, input_mode: RnnInputMode, direction_mode: DirectionMode, algorithm: RnnAlgorithm, hidden_size: i32, num_layers: i32, batch_size: i32) -> Result<Self::RC, Error> {
+        unimplemented!()
+    }
+
+    fn rnn_sequence_descriptors(&self,
+                                src: &SharedTensor<T>,
+                                sequence_length: i32,
+                                input_size: i32,
+                                hidden_size: i32,
+                                batch_size: i32,
+                                num_layers: i32)
+                                -> Result<RnnSequenceDescriptors, Error> {
+        unimplemented!()
```

Comment:
I prefer to add a little message so the user knows **what** has yet to be implemented :)